RFCs/2021-04-16-76-collection-policies.md

```diff
@@ -43,8 +43,9 @@ visor:
       handlers:
         # default configuration for the stream handlers
         config:
-          periods: 5
-          max_deep_sample: 50
+          periods: 2 #default is 5
+          max_deep_sample: 50 #default is 100
+          topn_size: 5 #default is 10
```

Comment:
let's stick with "count" instead of "size", as per the existing metrics member variable

Reply:
i've made `topn_count` here. Do you think that it should be `top_count` (without the 'n')

Reply:
topn_count is good